.travis.yml

```diff
@@ -12,14 +12,18 @@ before_deploy: |
   if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
     cabal install -f 'static'
     tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    curl https://github.com/lindell/github-release-cli/releases/download/1.3.0/github-releaser-travis -L --output github-releaser && chmod +x github-releaser
+    export GITHUB_OAUTH_TOKEN="$GITHUB_TRAVIS_DEPLOY"
+    export RELEASE_NAME=Dotenv-Release-v$(cabal exec dotenv -- --version)
```

Comment:
@CristhianMotoche what do you think about dropping the prefix here `Dotenv-Release-`, taking into account all releases are uploaded to the same repository, I would say there is no need to add the project name to the release, perhaps just the version?

Reply:
Oh, make total sense @sestrella :+1: